frontend/api/routes.py

```diff
@@ -26,6 +26,7 @@ def define_routes(application):
     application.route("/probes", callback=probes.list)
     # files routes
     application.route("/search/files", callback=search.files)
+    application.route("/files/<filesha256>", callback=search.get_file)
```

Comment:
I agree. Do I create a new `frontend.api.controllers`, maybe `files`?

And then, for instance:
* `files/{sha256}` - info on the file if it exists
* `files/{sha256}/get` - get back the file
* `files/{sha256}/names` - associated names

I'm not sure the `open(fobj.path).read()` is the better way to render a file, it forces the full read in memory before actually rendering it...